copy.go

```diff
@@ -49,7 +64,16 @@ func Copy(ctx context.Context, src Target, srcRef string, dst Target, dstRef str
 		return ocispec.Descriptor{}, err
 	}
 
-	if err := CopyGraph(ctx, src, dst, root); err != nil {
+	// TODO: manifest filter? 1. media type filter 2. platform filter
```

Comment:
The platform type filtering depends on the client, oras-go only need to provide the manifestFilter options.

For the mediatype, manifestFilter can do the manifest filtering but we still need downEdgeFilter support.

Reply:
Yeah this was a temporary note to remind myself.